Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -407,7 +407,7 @@ When using either `continueAfterFailure` or
 ``require(_:_:sourceLocation:)-5l63q``, other tests will continue to run after
 the failed test method or test function.
 
-#### Converting XCTestExpectation and XCTWaiter
+#### Convert XCTestExpectation and XCTWaiter
```

Comment:
Delete; can't have subheadings within a section. Also, don't include symbols in headings.

Reply:
As above, separate sections?